shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -161,10 +190,18 @@ - (instancetype)initWithPlugin:(NSString*)pluginKey flutterEngine:(FlutterEngine
 }
 
 - (NSView*)view {
-  if (!_flutterEngine.viewController.viewLoaded) {
-    [_flutterEngine.viewController loadView];
+  return [self viewForId:kFlutterDefaultViewId];
+}
+
+- (NSView*)viewForId:(uint64_t)viewId {
+  FlutterViewController* controller = [_flutterEngine viewControllerForId:viewId];
+  if (controller == nil) {
+    return nil;
   }
```

Comment:
Technically this isn't needed since `controller.viewLoaded` will return nil, `[controller loadView]` will be a no-op, and `controller.flutterView` will return nil... but I'm kind of tempted to say leave it to make the code easier to reason about anyway.

Reply:
Sorry. Are you suggesting me to keep the nil check of `controller` or not?